app/src/main/java/aero/minova/cas/app/extension/UserGroupUsersExtension.java

```diff
@@ -0,0 +1,114 @@
+package aero.minova.cas.app.extension;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+
+import aero.minova.cas.CustomLogger;
+import aero.minova.cas.api.domain.Column;
+import aero.minova.cas.api.domain.DataType;
+import aero.minova.cas.api.domain.Row;
+import aero.minova.cas.api.domain.SqlProcedureResult;
+import aero.minova.cas.api.domain.Table;
+import aero.minova.cas.api.domain.Value;
+import aero.minova.cas.app.util.ResponseEntityUtil;
+import aero.minova.cas.controller.SqlProcedureController;
+import aero.minova.cas.service.AuthoritiesService;
+import aero.minova.cas.service.AuthorizationService;
+import aero.minova.cas.service.UserGroupService;
+import aero.minova.cas.service.UsersService;
+import aero.minova.cas.service.model.Users;
+import jakarta.annotation.PostConstruct;
+
+@Component
+public class UserGroupUsersExtension {
+
+	@Autowired
+	protected SqlProcedureController sqlProcedureController;
+
+	@Autowired
+	protected CustomLogger logger;
+
+	@Autowired
+	protected AuthorizationService authorizationService;
+
+	@Autowired
+	protected UserGroupService userGroupService;
+
+	@Autowired
+	protected UsersService usersService;
+
+	@Autowired
+	protected AuthoritiesService authoritiesService;
+
+	@PostConstruct
+	public void setup() {
+		String formName = "UserGroupUsers";
+		sqlProcedureController.registerExtension("xpcasInsert" + formName, this::insert);
```

Comment:
Ist das nicht schon in der BaseExtension?

Dann kann man hier doch super.setup(formName); hinschreiben, oder?